presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestMergeIdenticalWindows.java

```diff
@@ -319,6 +320,54 @@ public void testNotMergeAcrossJoinBranches()
                                                 anyTree())))));
     }
 
+    @Test
+    public void testNotMergeDifferentOrdering()
+    {
+        @Language("SQL") String sql = "select " +
+                "sum(extendedprice) over (partition by suppkey order by orderkey rows between unbounded preceding and current row) sum_extendedprice_A, " +
+                "sum(quantity) over (partition by suppkey order by orderkey desc rows between UNBOUNDED preceding and CURRENT ROW) sum_quantity_C, " +
```

Comment:
But this test is new. You don't need to introduce such a inconsistency to fix it in other places.